aspnetcore/fundamentals/logging/index.md

```diff
@@ -743,6 +833,29 @@ The default location for log files is in the *D:\\home\\LogFiles\\Application* f
 
 The provider only works when the project runs in the Azure environment. It has no effect when the project is run locally&mdash;it doesn't write to local files or local development storage for blobs.
 
+### Azure log streaming
+
+Azure log streaming lets you view log activity in real time from:
+
+* The application server
+* The web server
+* Failed request tracing
+
+To configure Azure log streaming:
+
+* Navigate to the **Diagnostics Logs** page from your application's portal page
+* Set **Application Logging (Filesystem)** to on.
+
+![Azure portal diagnostic logs page](index/_static/azure-diagnostic-logs.png)
+
+Navigate to the **Log Streaming** page to view application messages. They're logged by the application through the `ILogger` interface.
+
+![Azure portal application log streaming](index/_static/azure-log-streaming.png)
+
+### Azure Application Insights trace logging
+
+The [Application Insights](https://azure.microsoft.com/services/application-insights/) SDK is capable of collecting trace telemetry from logs generated via the ASP.NET Core logging infrastructure. For more information, see [Application Insights for ASP.NET Core](https://docs.microsoft.com/azure/application-insights/app-insights-asp-net-core) and [Microsoft/ApplicationInsights-aspnetcore Wiki: Logging](https://github.com/Microsoft/ApplicationInsights-aspnetcore/wiki/Logging).
```

Comment:
- Consider using the following link to keep the reader in the docs: /azure/azure-monitor/.
- Prefer a relative link for docs links instead of providing the fully qualified URL.